### PR TITLE
Dockerfile 1.6

### PR DESCRIPTION
**Pull request by pierallard**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+RUN pip install sphinx~=1.5.3
+RUN pip install git+https://github.com/fabpot/sphinx-php.git && \
+    pip install git+https://github.com/mickaelandrieu/sphinxcontrib.youtube.git
+
+# Akeneo PIM installation
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && \
+    apt-get install -y wget unzip && \
+    apt-get install -y php5 mysql-server apache2 libapache2-mod-php5 php5-cli php5-apcu php5-mcrypt php5-intl php5-mysql php5-curl php5-gd mongodb php5-mongo && \
+    apt-get clean && \
```
Comment (damien-carcel) [inline Dockerfile]: You can do better to clean and minimize the image size:
https://github.com/akeneo/Dockerfiles/blob/master/php/Dockerfile#L11

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        rm /tmp/1.6.zip
+    else
+        echo "Rebuild Akeneo PIM assets..."
+        rm -rf /home/akeneo/pim-docs/1.6.zip
+        mv /tmp/1.6.zip /home/akeneo/pim-docs/1.6.zip
+        rm -rf /home/akeneo/pim-docs/pim-community-dev-1.6
+        unzip /home/akeneo/pim-docs/1.6.zip -d /home/akeneo/pim-docs/
+        cd /home/akeneo/pim-docs/pim-community-dev-1.6/ && php -d memory_limit=3G ../composer.phar install --no-dev --no-suggest
+        service mysql start && \
+        cd /home/akeneo/pim-docs/pim-community-dev-1.6/ && php app/console pim:installer:assets --env=prod
```
Comment (docteurklein) [inline build.sh]: why install assets twice ? (in the image already)

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+RUN pip install --upgrade pip
+RUN pip install sphinx~=1.5.3
+RUN pip install git+https://github.com/fabpot/sphinx-php.git && \
+    pip install git+https://github.com/mickaelandrieu/sphinxcontrib.youtube.git
+
+# Akeneo PIM installation
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && \
+    apt-get install -y wget unzip && \
+    apt-get install -y php5 mysql-server apache2 libapache2-mod-php5 php5-cli php5-apcu php5-mcrypt php5-intl php5-mysql php5-curl php5-gd mongodb php5-mongo && \
```
Comment (docteurklein) [inline Dockerfile]: it amazes me (:face_with_head_bandage: ) that we need php and the PIM, and mysql installed to build the docs

Comment (damien-carcel) [reply Dockerfile]: :trollface:

File: build.sh (hunk 2 above)
Comment (pierallard) [reply build.sh]: Answered IRL

**Pull request closed without merging**